modules/media/README.md

```diff
@@ -1,51 +1,57 @@
-## Media Module
+# Media
 
-### 📄 Overview
+## Purpose
 
-Media module allows users to **upload**, **search** and **edit** media files associated with a specific candidate timepoint in Loris.
-Any kind of data associated with a candidate timepoint can be uploaded through this module: PDFs, videos, recordings, scripts, log files, etc. Files can optionally be associated to a specific instrument form within a given candidate timepoint.
+Media module allows users to **upload**, **search** and **edit** media files 
+associated with a specific candidate timepoint in Loris. Any kind of data 
+associated with a candidate timepoint can be uploaded through this module: 
+PDFs, videos, recordings, scripts, log files, etc. Files can optionally be 
+associated to a specific instrument form within a given candidate timepoint.
 
+## Intended Users
 
->Note: Currently editing functionality only allows editing of certain metadata fields, such as `Comments` and `Date of Administration`.
+The Media module is used by data entry staff to upload candidate and timepoint 
+specific media. The module can also be used by 
```

Comment:
can also be used by??

Reply:
right back at you... lol anyone i'm missing ?